gradle/libs.versions.toml

```diff
@@ -0,0 +1,89 @@
+
+tools-common = { module = "com.android.tools:common", version.ref = "tools-common" }
+tools-layoutlib = { module = "com.android.tools.layoutlib:layoutlib-api", version.ref = "tools-layoutlib" }
+tools-ninepatch = { module = "com.android.tools:ninepatch", version.ref = "tools-ninepatch" }
+tools-sdkCommon = { module = "com.android.tools:sdk-common", version.ref = "tools-sdkCommon" }
+
+androidx-annotations = { module = "androidx.annotation:annotation", version.ref = "androidx-annotations" }
+androidx-compose-compiler = { module = "androidx.compose.compiler:compiler", version.ref = "androidx-compose-compiler" }
+androidx-compose-ui-material = { module = "androidx.compose.material:material", version.ref = "androidx-compose-ui-material" }
+androidx-lifecycle-common = { module = "androidx.lifecycle:lifecycle-common", version.ref = "androidx-lifecycle" }
+
+kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
+kotlinx-coroutines-core = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version.ref = "kotlin-coroutines" }
+
+guava = { module = "com.google.guava:guava", version.ref = "guava" }
+kxml2 = { module = "kxml2:kxml2", version.ref = "kxml2" }
+okio = { module = "com.squareup.okio:okio", version.ref = "okio" }
+
+moshi-adapters = { module = "com.squareup.moshi:moshi-adapters", version.ref = "moshi" }
+moshi-core = { module = "com.squareup.moshi:moshi", version.ref = "moshi" }
+moshi-kotlin = { module = "com.squareup.moshi:moshi-kotlin-codegen", version.ref = "moshi" }
+
+jcodec-core = { module = "org.jcodec:jcodec", version.ref = "jcodec" }
+jcodec-javase = { module = "org.jcodec:jcodec-javase", version.ref = "jcodec" }
+bytebuddy-agent = { module = "net.bytebuddy:byte-buddy-agent", version.ref = "bytebuddy" }
+bytebuddy-core = { module = "net.bytebuddy:byte-buddy", version.ref = "bytebuddy" }
+
+junit = { module = "junit:junit", version.ref = "junit" }
+testParameterInjector = { module = "com.google.testparameterinjector:test-parameter-injector", version.ref = "testParameterInjector" }
+truth = { module = "com.google.truth:truth", version.ref = "truth" }
+assertj = { module = "org.assertj:assertj-core", version.ref = "assertj" }
+
+plugin-android = { module = "com.android.tools.build:gradle", version.ref = "agp" }
+plugin-dokka = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version.ref = "kotlin" }
+plugin-grgit = { module = "org.ajoberstar.grgit:grgit-gradle", version.ref = "grgit" }
+plugin-kotlin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
+plugin-mavenPublish = { module = "com.vanniktech:gradle-maven-publish-plugin", version.ref = "mavenPublish" }
+plugin-spotless = { module = "com.diffplug.spotless:spotless-plugin-gradle", version.ref = "spotless" }
+plugin-versions = { module = "com.github.ben-manes:gradle-versions-plugin", version.ref = "gradle-versions" }
+
+[plugins]
+# No plugins, because ?, see plugin-* under [libraries].
```

Comment:
delete

Reply:
why are we not using plugins? I think this would be a place to answer this question.

Reply:
Except maybe for the documented and recommended world-wide Gradle conventions of using plugins+alias with Version Catalogs. 😅

Anyone outside of cashapp will probably question this...